src/traits/storage/memory_storage.rs

```diff
@@ -150,8 +149,8 @@ mod test {
     impl NodeType for DummyTypes {
         type Time = ViewNumber;
         type BlockType = DummyBlock;
-        type SignatureKey = Ed25519Pub;
-        type VoteTokenType = StaticVoteToken<Ed25519Pub>;
+        type SignatureKey = BN254Pub;
```

Comment:
I think we should use another naming here, something like BLSVerKey (see https://github.com/EspressoSystems/jellyfish/blob/903d32a4a4c65a1e41fc9f2da602e323779659e0/primitives/src/signatures/bls_over_bn254.rs#L239C1-L239C1) as we are refering to the public key of some signature scheme. Also BN254 is not specific enough as there are two possible groups.

Reply:
I made an issue for this here: https://github.com/EspressoSystems/HotShot/issues/1592

Reply:
Thanks!